apps/frontend/src/app/poll/table/table.component.html

```diff
@@ -24,7 +24,8 @@
               id="name"
               placeholder="Your name"
               [disabled]="poll.settings.anonymous"
-              [(ngModel)]="newParticipant.name" (change)="validateNew()"
+              [(ngModel)]="newParticipant.name"
+              (input)="validateNew()"
```

Comment:
Change is only called when input loses focus, therefore I had to adjust this with using an input event.